storybook/src/data-display/DataTable.mdx

```diff
@@ -111,6 +111,67 @@ Here is an example utilizing the `useMemo` hook. This specific logic is applied
 
 Now you have a DataTable component with sorting enabled for the specified column, and you can implement your own sorting logic based on your application requirements.
 
+## Multi-column Sorting in DataTable
+
+The `DataTable` component in `@tiller-ds` empowers you to implement **multi-column sorting**, allowing users to sort data across multiple columns simultaneously.
+This enhanced functionality is achieved through the `toggleSortBy` method provided by the `useTable` hook.
+
+To enable multi-column sorting, follow these steps:
+
+**1.** Ensure that the columns for which you want to implement sorting have the `canSort` prop set to `true`:
+```tsx
+<DataTable.Column header="Name" accessor="name" canSort={true} />
+<DataTable.Column header="Surname" accessor="surname" canSort={true} />
+```
+
+**2.** For multi-column sorting, apply sorting logic based on the priority of columns specified in the sort state:
+```tsx
+  const [dataTableState, dataTableHook] = useDataTable({
+    defaultSortBy: [
+      {
```

Comment:
Multiple columns should be defined here, right?

Reply:
No, you don't need to explicitly define multiple columns in the defaultSortBy prop when you have enableMultiSort enabled for the DataTable. The defaultSortBy prop is used to set the initial sorting configuration when the table loads. If you've set canSort to true for the desired columns and enabled multi-column sorting, users can dynamically sort the table by clicking on the column headers.